presto-docs/README.md

```diff
@@ -10,6 +11,32 @@ The presto-docs module contains the reference documentation for Presto.
 - [Versioning](#versioning)
 - [Known Issues](#known-issues)
 
+## Writing and contributing
+
+We welcome any contributions to the documentation. Contributions need to [follow
+the same process as code contributions](https://prestosql.io/development/) and
+can be part of your code contributions or separate documentation improvements.
+
+The documentation follows the Google developer documentation style guide for any
+new documentation:
+
+- [Google developer documentation style guide](https://developers.google.com/style)
```

Comment:
I can add more links to specific pages.. this is just a list of items that writer often get wrong..